src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ConnectionStringKeys.cs

```diff
@@ -119,9 +124,22 @@ public class ConnectionStringKeys
         /// <summary>
         /// Endpoint to use for AAD authentication
         /// </summary>
-        public const string AADAuthenticationEndpointKey = "AADAuthEndpoint";
+        public const string AADAuthUriKey = "AADAuthUri";   //Most probably ActiveDirectoryAuthority
```

Comment:
why are we changing this value?  let's allow people to continue using the old vealue for this.